README.md

```diff
@@ -50,9 +50,9 @@ buildscript {
 
 ## Examples
 
-A stand-alone example project is located under [exampleProject]
-(https://github.com/google/protobuf-gradle-plugin/tree/master/exampleProject).
-Run `../gradlew build` under that directory to test it out.
+A stand-alone example projects are located under [samples]
```

Comment:
I'd prefer naming the directory "examples", to follow the same pattern as other code repos.

Also, since the groovy DSL is still the default for gradle, I think it's fine to keep the name "exampleProject".